features/bevragen/raadpleeg-met-burgerservicenummer/fout-cases.feature

```diff
@@ -0,0 +1,131 @@
+#language: nl
+
+Functionaliteit: Raadpleeg met burgerservicenummer - fout cases
```

Comment:
Moeten er hier niet ook scenario's opgenomen worden met een leeg of foutief 'gemeenteVanInschrijving'?